target/linux/kirkwood/patches-5.4/203-blackarmor-nas220.patch

```diff
@@ -0,0 +1,92 @@
+--- a/arch/arm/boot/dts/kirkwood-blackarmor-nas220.dts
++++ b/arch/arm/boot/dts/kirkwood-blackarmor-nas220.dts
+@@ -17,6 +17,13 @@
+ 	compatible = "seagate,blackarmor-nas220","marvell,kirkwood-88f6192",
+ 		     "marvell,kirkwood";
+ 
++	aliases {
++		led-boot = &led_fault;
++		led-failsafe = &led_fault;
++		led-running = &led_healthy;
++		led-upgrade = &led_fault;
++	};
++
+ 	memory { /* 128 MB */
+ 		device_type = "memory";
+ 		reg = <0x00000000 0x8000000>;
+@@ -36,14 +43,14 @@
+ 		compatible = "gpio-keys";
+ 
+ 		reset {
+-			label = "Reset";
+-			linux,code = <KEY_POWER>;
++			label = "Reset Button";
++			linux,code = <KEY_RESTART>;
+ 			gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
+ 		};
+ 
+-		button {
+-			label = "Power";
+-			linux,code = <KEY_SLEEP>;
++		power {
++			label = "Power Button";
++			linux,code = <KEY_POWER>;
+ 			gpios = <&gpio0 26 GPIO_ACTIVE_LOW>;
+ 		};
+ 	};
+@@ -51,11 +58,27 @@
+ 	gpio-leds {
+ 		compatible = "gpio-leds";
+ 
+-		blue-power {
++		led_power: power {
+ 			label = "nas220:blue:power";
+ 			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+ 			linux,default-trigger = "default-on";
+ 		};
++
++		disk {
++			label = "nas220:blue:disk";
++			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
++			linux,default-trigger = "disk-activity";
++		};
++
++		led_healthy: healthy {
```

Comment:
Please keep the label and the node name consistent, i.e. name this
led_status_blue: status_blue {

Same for the following LED.